cfgov/jinja2/v1/_includes/organisms/filterable-list.html

```diff
@@ -150,14 +150,10 @@
                             <div class="content-l_col
                                         content-l_col-1-2">
                                 {% set post_url = pageurl(post) %}
-                                {% set heading = '<a href="' ~ post_url ~ '"><h2 class="h3">' ~
+                                {% set heading = '<h2 class="h3">' ~
                                                 post.preview_title ~
-                                                '</h2></a>'
+                                                '</h2>'
```

Comment:
Could you indent these two lines one more space to line up with the line above?

Reply:
Fixed in 9878c52.